scripts/test_nearlib.sh

```diff
@@ -2,31 +2,47 @@
 set -ex
 
 ./scripts/requirements_check.sh
+repo_dir=$(realpath -e "$(dirname "${BASH_SOURCE[0]}")/..")
 
-export RUST_BACKTRACE=full
-./scripts/start_unittest.py --local &
+# Builds and starts `neard`, assumes it's running in ${repo_dir}'
+# May fail silently, leading to hard to debug CI failures
+# TODO: add option to build only and run checks, this will allow us to see those errors synchronously.
+
+# TODO: This test leaves `near` process running even after the test is over
+# pkill near || echo "Nothing to kill"
```

Comment:
This probably should be rewritten in Python and then we can handle killing child processes with no issues.